src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -709,6 +679,19 @@ void CC32xxWiFi::run_on_network_thread(std::function<void()> callback)
 }
 
 /*
+ * CC32xxWiFi::fd_remove()
+ */
+void CC32xxWiFi::fd_remove(int16_t socket)
```

Comment:
Please add documentation to the slSockets array. what is it indexed by? fd? sd? what is the value stored? fd? sd?